.buildkite/scripts/steps/functional/apm_cypress.sh

```diff
@@ -12,8 +12,8 @@ APM_CYPRESS_RECORD_KEY="$(retry 5 5 vault read -field=CYPRESS_RECORD_KEY secret/
 export JOB=kibana-apm-cypress
 IS_FLAKY_TEST_RUNNER=${CLI_COUNT:-0}
 
-# Disable parallel tests and dashboard recording when running them in the flaky test runner
-if [[ "$IS_FLAKY_TEST_RUNNER" -ne 1 ]]; then
+#Enabling cypress dashboard recording when PR is labeled with `apm:cypress-record` and we are not using the flaky test runner
```

Comment:
I guess if the label is required now it won't run on the flaky test runner anyways, so the condition is no longer needed but maybe it's good to leave it so it's explicit / we don't forget?

Reply:
I think we can keep it.